Unable to check updates for K-9 mail

**Important first questions**

1. Is your FFUpdater up-to-date? If not, please abort and update FFUpdater.

2. Do you spot similar issues in https://github.com/Tobi823/ffupdater/issues? If yes, please abort and add your error message as a comment to the existing issue.

**Describe the bug**
Replace this paragraph with a clear and concise description of what the bug is.

Other apps update fine, but K-9 mail always says "temporary network error"

**To Reproduce**
Please insert the steps to reproduce the behavior here:
1. Open the app
2. Check updates for K-9 mail
3. ...

**Expected behavior**
Please give a clear and concise description of what you expected to happen.

I expected to see the installed and available version, instead I see network error

**Error message from FFUpdater**
Please copy and paste the error message with the stacktrace and error log here:

Stacktrace:
```
de.marmaro.krt.ffupdater.DisplayableException: Unable to fetch the latest update for K9MAIL.
	at de.marmaro.krt.ffupdater.app.impl.base.InstalledAppStatusFetcher$DefaultImpls.findStatusAndCacheIt(InstalledAppStatusFetcher.kt:57)
	at de.marmaro.krt.ffupdater.app.impl.base.InstalledAppStatusFetcher$DefaultImpls.access$findStatusAndCacheIt(InstalledAppStatusFetcher.kt:17)
	at de.marmaro.krt.ffupdater.app.impl.base.InstalledAppStatusFetcher$findStatusAndCacheIt$1.invokeSuspend(InstalledAppStatusFetcher.kt:0)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at android.os.Handler.handleCallback(Handler.java:959)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loopOnce(Looper.java:232)
	at android.os.Looper.loop(Looper.java:317)
	at android.app.ActivityThread.main(ActivityThread.java:8705)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:886)
Caused by: de.marmaro.krt.ffupdater.network.exceptions.InvalidApiResponseException: can't find latest release
	at de.marmaro.krt.ffupdater.network.github.GithubConsumer.findLatestRelease(GithubConsumer.kt:30)
	at de.marmaro.krt.ffupdater.network.github.GithubConsumer$findLatestRelease$1.invokeSuspend(GithubConsumer.kt:0)
	... 10 more

```
Logs:
```
de.marmaro.krt.ffupdater.DisplayableException: Unable to fetch the latest update for K9MAIL.
	at de.marmaro.krt.ffupdater.app.impl.base.InstalledAppStatusFetcher$DefaultImpls.findStatusAndCacheIt(InstalledAppStatusFetcher.kt:57)
	at de.marmaro.krt.ffupdater.app.impl.base.InstalledAppStatusFetcher$DefaultImpls.access$findStatusAndCacheIt(InstalledAppStatusFetcher.kt:17)
	at de.marmaro.krt.ffupdater.app.impl.base.InstalledAppStatusFetcher$findStatusAndCacheIt$1.invokeSuspend(InstalledAppStatusFetcher.kt:0)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at android.os.Handler.handleCallback(Handler.java:959)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loopOnce(Looper.java:232)
	at android.os.Looper.loop(Looper.java:317)
	at android.app.ActivityThread.main(ActivityThread.java:8705)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:886)
Caused by: de.marmaro.krt.ffupdater.network.exceptions.InvalidApiResponseException: can't find latest release
	at de.marmaro.krt.ffupdater.network.github.GithubConsumer.findLatestRelease(GithubConsumer.kt:30)
	at de.marmaro.krt.ffupdater.network.github.GithubConsumer$findLatestRelease$1.invokeSuspend(GithubConsumer.kt:0)
	... 10 more

```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 79.2.5 (175) release Github |
| Device | Pixel 6a (bluejay, bluejay, bluejay) |
| Manufacturer | google (Google) | 
| Supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| Android version | 15 (SDK: 35) |
| OS | r-68354e84479a5a03-12mk, android-build, release-keys, 1726249029000 |

--- a/ffupdater/src/main/java/de/marmaro/krt/ffupdater/app/impl/K9Mail.kt
+++ b/ffupdater/src/main/java/de/marmaro/krt/ffupdater/app/impl/K9Mail.kt
@@ -39,7 +39,7 @@ object K9Mail : AppBase() {
         val result = GithubConsumer.findLatestRelease(
             repository = GithubConsumer.GithubRepo("thunderbird", "thunderbird-android", 0),
             isValidRelease = { true },
-            isSuitableAsset = { it.name.matches("k9-([0-9.]+).apk".toRegex()) },
+            isSuitableAsset = { it.name.matches("k9mail-([0-9.]+).apk".toRegex()) },
             requireReleaseDescription = false,
         )
         return LatestVersion(

